src/adapters/lokijs/worker/lokiExtensions.js

```diff
@@ -1,39 +1,79 @@
 // @flow
+/* eslint-disable no-undef */
 
 import Loki, { LokiMemoryAdapter } from 'lokijs'
-import LokiIndexedAdapter from 'lokijs/src/loki-indexed-adapter'
 
-export function newLoki(name: ?string, peristenceAdapter?: LokiMemoryAdapter): Loki {
-  const newAdapter =
-    process.env.NODE_ENV === 'test' ? new LokiMemoryAdapter() : new LokiIndexedAdapter(name)
+const isIDBAvailable = () => {
+  return new Promise(resolve => {
+    // $FlowFixMe
+    if (typeof indexedDB === 'undefined') {
```

Comment:
What's the problem with Flow?

Reply:
it doesn't know that `indexedDB` is defined, and I can't do `window.indexedDB` because this code can run either in normal context (has window) or in a web worker (doesn't have window)

Reply:
Ok, we can leave it as is :)